### PR TITLE
[service] Add API header for query_client service

### PR DESCRIPTION
**Pull request by anyj0527**
- Add signatures of two APIs for query_client service

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

### CODE REVIEW

File: c/include/ml-api-service.h [hunk 1]
```diff
@@ -190,6 +190,37 @@ int ml_service_destroy (ml_service_h handle);
...
+ * @param[in] option The option handle used for creating query client.
+ * @param[out] handle Newly created service handle is returned.
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ */
+int ml_service_query_create_client (ml_option_h option, ml_service_h *handle);
```
Comment (jaeyun-jung) [inline c/include/ml-api-service.h:206]: Discussion: ml_service_query_create() vs ml_service_query_create_client()
(e.g. service_query_create() > service_query_request() > service_destroy())

Comment (anyj0527) [reply c/include/ml-api-service.h:206]: `ml_service_query_create()` seems good to me.

File: c/include/ml-api-service.h [hunk 2]
```diff
@@ -190,6 +190,37 @@ int ml_service_destroy (ml_service_h handle);
...
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ */
+int ml_service_query_create (ml_option_h option, ml_service_h *handle);
+
+/**
+ * @brief Requests query service an output with given input data.
```
Comment (wiertel) [inline c/include/ml-api-service.h:209]: It's technically correct, but might sound better like this "Requests the query service to process the @a input tensor and produce an @a output tensor." OR "Requests the query service to compute @output data from the given @a input." (the word "compute" can be replaced by something more appropriate for ML)

Comment (anyj0527) [reply c/include/ml-api-service.h]: Thanks! I changed the sentence with your concise suggestion, PTAL:
"@brief Requests the query service to process the @a input and produce an @a output."

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**